Src/WitsmlExplorer.Frontend/components/ContentViews/EditInterval.tsx

```diff
@@ -29,6 +34,35 @@ const EditInterval = (): React.ReactElement => {
       payload: { logCurveInfo: logCurveInfoWithUpdatedIndex }
     });
   };
+
+  const validate = (current: string) => {
+    return parse(current, dateTimeFormat, new Date());
+  };
+  const isTimeCurve = () => {
+    return curveType === "Time";
```

Comment:
It's a bad idea to use the name of the curve to determine if it's a time log as it can have any name. It's better to check the indexType of the selectedObject:

`return selectedLog?.indexType === WITSML_INDEX_TYPE_DATE_TIME`

You need to also bring in the object:
```
const { selectedObject, selectedLogCurveInfo } = navigationState;
const selectedLog = selectedObject as LogObject
```

Reply:
fixed